### PR TITLE
adding Text Pane

### PR DESCRIPTION
**Pull request by soumith**
adds a simple pane that can display text.
screen shot 2015-09-11 at 6 30 51 pm

### CODE REVIEW

File: static/panes.js [hunk 1]
```diff
@@ -549,6 +549,24 @@ PlotPane.prototype = extend(Object.create(Pane.prototype), {
   },
 });
 
+function TextPane(id) {
+  Pane.call(this, id);
+
+    var self = this
+    , content = this.content;
```
Comment (szym) [inline static/panes.js]: nit: indent

File: static/panes.js [hunk 2]
```diff
@@ -614,3 +637,5 @@ function load() {
 on(document, 'DOMContentLoaded', load);
 
 }).call(window);
+
+
```
Comment (szym) [inline static/panes.js]: nit: extra whitespace

**Comment by soumith** [comment]
thanks, fixed the nits.

**Comment by szym** [comment]
Thanks for the contribution.

**Pull request merged**